base/src/main/java/com/smartdevicelink/proxy/rpc/GetVehicleData.java

```diff
@@ -540,21 +541,38 @@ public void setCloudAppVehicleID(boolean cloudAppVehicleID){
     public Boolean getCloudAppVehicleID(){
         return getBoolean(KEY_CLOUD_APP_VEHICLE_ID);
     }
-
-    /**
-     * Sets a boolean value for OEM Custom VehicleData.
-     * @param vehicleDataName a String value
-     * @param vehicleDataState a boolean value
-     */
-    public void setOEMCustomVehicleData(String vehicleDataName, Boolean vehicleDataState){
-        setParameters(vehicleDataName, vehicleDataState);
-    }
-
-    /**
-     * Gets a boolean value for OEM Custom VehicleData.
-     * @return a Boolean value.
-     */
-    public Boolean getOEMCustomVehicleData(String vehicleDataName){
-        return getBoolean(vehicleDataName);
-    }
-}
+
+    /**
+     * Sets a boolean value for OEM Custom VehicleData.
+     * @param vehicleDataName a String value
+     * @param vehicleDataState a boolean value
+     */
+    public void setOEMCustomVehicleData(String vehicleDataName, Boolean vehicleDataState){
+        setParameters(vehicleDataName, vehicleDataState);
+    }
+
+    /**
+     * Gets a boolean value for OEM Custom VehicleData.
+     * @return a Boolean value.
+     */
+    public Boolean getOEMCustomVehicleData(String vehicleDataName){
+        return getBoolean(vehicleDataName);
+    }
+
+    /**
+     * Sets a status value for WindowStatus.
+     * @param status a Boolean value
+     */

```

Comment:
```suggestion
    /**
     * Sets the windowStatus.
     *
     * @param windowStatus See WindowStatus
     * @since SmartDeviceLink 7.0.0
     */
```
Please use the [RPC generator](https://github.com/smartdevicelink/sdl_java_suite/tree/master/generator) in the sdl_java_suite project to ensure the RPC matched the spec exactly. The generator can also be used to update the XML comments at the top of this file